src/js/src/App.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div id="app">
-    <router-view />
+    <!-- En enforce component SolrWayback update when moving from '/' to '/search' -->
+    <router-view :key="$route.path" />
```

Comment:
This will trigger complete re-rendering and rerun of life-cycle hooks when routing - but I guess that this is actually what you want :)